tests/scalapack/scalapack_15.cc

```diff
@@ -94,8 +94,7 @@ test(const unsigned int size,
     NumberType vl = 0, vu = 0;
     int        il = 1, iu = size;
     char       sign = 'S';
-    NumberType abstol;
-    lamch(&sign, abstol);
+    NumberType abstol lamch(&sign);
```

Comment:
Should this be
```suggestion
    NumberType abstol = lamch(&sign);
```
like in the previous statement?

Reply:
Of course! Fixed.